### PR TITLE
upgrade to ipmitool-xcat-1.8.15

### PR DESCRIPTION
**Pull request by chenglch**
saneretry.patch is useful for ipmitool-1.8.15,
eventfix and spdfix have already been patched.
add rflash patch to delete upgrade prompt question.

### CODE REVIEW

**Comment by whowutwut** [comment]
@tingtli How do we build this package, I thought the `builddeps` is executed from the xcat-core?   @daniceexi  Not sure who should review this, I don't have any experience with this package...

**Comment by daniceexi** [comment]
@whowutwut Generally the dependency rpm packages are build by ourselves manually and put it in the GSA directory, then builddeps script just generate repo data and tar them to a tar.gz file.
@jjohnson42 should be the correct one to review this change.

**Comment by daniceexi** [comment]
@whowutwut 
I saw that we did not add team operation permission for xcat-dep. I think it should be OK to add **Developers** team has permission to access 'xcat-dep' repo.

**Comment by whowutwut** [comment]
Done, added the xcat-dep repo to the Developers Team.

**Comment by whowutwut** [comment]
I think no on follow this one,  Should we merge and then how is this built?  @tingtli , can you look at my 2nd comment above?  Is will this be broken for ESP?

**Comment by whowutwut** [comment]
@jjohnson42  could you review these changes?

**Comment by whowutwut** [comment]
@daniceexi  who will build the ipmitool-xcat RPM?  Do we need to define the options that we need to build with... for example, specify the `--enable-intf-lanplus` ?

**Comment by jjohnson42** [comment]
Given that the reasons for ipmitool-xcat on Lenovo systems are either fixed, or just as well addressed with confluent baked-in SOL, I have no objections.

**Comment by daniceexi** [comment]
@zet809 could you review this change and make sure it can be merged ASAP so that the test for this item could be started soon in next week.

File: ipmitool/ipmitool.spec [hunk 1]
```diff
@@ -1,14 +1,13 @@
 Name:         ipmitool-xcat
 Summary:      ipmitool - Utility for IPMI control
-Version:      1.8.11
+Version:      1.8.15
 Release:      3
 License:      BSD
 Group:        Utilities
 Packager:     Jarrod Johnson <jbjohnso@us.ibm.com>
```
Comment (zet809) [inline ipmitool/ipmitool.spec]: Pls modify Packager info as to "IBM Corp."

**Comment by chenglch** [comment]
@zet809 , Thanks for your review, I have updated this patch to cover your concerns.

**Comment by zet809** [comment]
The code looks good to me, I agree to merge it.

**Pull request merged**